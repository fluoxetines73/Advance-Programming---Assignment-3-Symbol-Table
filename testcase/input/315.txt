BEGIN
PRINT
PRINT
BEGIN
PRINT
PRINT
INSERT e_123_2r string
RPRINT
PRINT
RPRINT
INSERT c number
RPRINT
ASSIGN c 1001
LOOKUP c
PRINT
END
END