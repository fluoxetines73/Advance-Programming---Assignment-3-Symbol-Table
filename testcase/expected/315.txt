



success
e_123_2r//2
e_123_2r//2
e_123_2r//2
success
c//2 e_123_2r//2
success
2
e_123_2r//2 c//2